migration/ament_tools.rst

```diff
@@ -91,6 +91,7 @@ ament test_results
 Behavioral changes
 ------------------
 
+The colcon ``test`` verb performs only the action of running tests. It does not configure, build or install targets.
```

Comment:
Wrap each sentence.

Add an empty line between the following content to create a separate paragraph?

Reply:
Oh yeah it was a signle sentend and I forgot to wrap when splitting it up :|

Empty line added as well. We could add some `verb` subsections so that the behavorial change sections is more readable as we add content (currently all items affect the test verb so I didnt add it)